isofit/utils/downloads.py

```diff
@@ -0,0 +1,101 @@
+"""
+Implements the `isofit download` subcommands
+"""
+
+import io
+import os
+import zipfile
+from functools import partial
+
+import click
+import requests
+from github import Github
+
+import isofit
+
+
+def downloadRelease(repo, output, name):
+    """
+    Downloads a specific tagged release from a given repository
+
+    Parameters
+    ----------
+    repo: str
+        Repository URL
+    output: str
+        Directory to download to
+    name: str
+        Name of the directory. The downloaded directory is based off the Github name,
+        so this is used to rename it
+    """
+    git = Github()
+    repo = git.get_repo(repo)
+    latest = repo.get_latest_release()
+    url = latest.zipball_url
+
+    click.echo(f"Downloading {url}")
+
+    req = requests.get(url)
+    with zipfile.ZipFile(io.BytesIO(req.content)) as zip:
+        temp = zip.infolist()[0].filename
+        zip.extractall(output)
+
+    os.rename(f"{output}/{temp}", f"{output}/{name}")
+    click.echo(f"Now available at: {output}/{name}")
+
+
+# Main command
+
+
+@click.group(invoke_without_command=True)
```

Comment:
@jammont @unbohn I don't think you want `invoke_without_command=True` either. That lets you run `$ isofit download` without a sub-command, which in this case does nothing. The `download` group is just a namespace, not a command.

Reply:
Fair enough! Wasn't sure if `isofit download --help` would work without it and modeled it off the `isofit.__init__` implementation. It'll be removed in the next PR.